docs/mkdocs.yml

```diff
@@ -0,0 +1,56 @@
+site_name: Couchbase Plugin
+theme:
+  name: material
+  custom_dir: 'material/'
+  palette:
+    primary:
+    accent:
+  logo: 'image/delphix-logo-white.png'
+  favicon: 'image/logo.png'
+  font:
+    text: Helvetica Neue
+    code: Ubuntu Mono
+
+copyright: Copyright &copy; 2020 Delphix Corp.
+
+google_analytics:
+  - 'UA-35429885-3'
+  - 'auto'
+
+repo_url: https://github.com/delphix/couchbase-plugin
+repo_name: Github
+use_directory_urls: false
+pages:
+  - Overview: 'index.md'
+  - Installation: 'Installation.md'
+  - PluginDevelopment : 'PluginDevelopment.md'
```

Comment:
Is it possible to have a space in between the words? "Plugin Development"?

Reply:
Yes, adding extra space in `PluginDevelopment`